### ISSUE TITLE
clicking on stylesheet links should open the corresponding file in a new tab

### ISSUE THREAD

**Issue by AnthonyLenglet** [microsoft/vscode-edge-devtools#100]
The link on the right of this image should open the stylesheet itself in a new tab

![image](https://user-images.githubusercontent.com/36004334/64773865-68dfa580-d553-11e9-94e5-c8be05881330.png)

The path on hover is *almost* correct, the only problem is that it starts with "localhost:4200/", everything after that is the correct path, so it seems like it wouldn't be tough to get the path itself

**Comment by JamesGreenaway** [comment]
@jalissia do you know if this works on macOS edge? I can't for the life of me get it to work on this end.

**Comment by jalissia** [comment]
@JamesGreenaway hmm, it should. What version/channel of Edge are you using? @codepo8 @vidorteg any known issues here?

**Comment by JamesGreenaway** [comment]
Version 79.0.309.65 (Official build) (64-bit)

**Comment by jalissia** [comment]
If you hover over the link, what path is it giving you? How close is it from the actual location of the file on disk?
Also what are you pathMapping setting are you using?

**Comment by JamesGreenaway** [comment]
If I hover over the link in any project it gives me the right path: http://localhost:8000/style.css
Any pathMapping setting seems to make no difference. When I click the link nothing happens in vscode.

**Comment by jalissia** [comment]
Right now the extension doesn't dynamically load files from outside your workspace folder. Do you have style.css on disk somewhere?

If so I would have expected pathMapping to work when it points at that location, something like:
```
"pathMapping": {
   "/": "${workspaceFolder}/my_styles/"
}
```
assuming `style.css` is located in your workspace in a subfolder named `my_styles`

**Comment by JamesGreenaway** [comment]
yep it's within the same workspace, in the same format you described.

**Comment by JamesGreenaway** [comment]
Can anybody confirm if they can get this working on macOS?

**Comment by GriffinLedingham** [comment]
I am experiencing the same issue on macOS, Edge 79.0.309.65 (Official build) (64-bit) as well as Version 80.0.361.32 (Official build) Beta (64-bit). Clicking common_style here just has the element panel highlight the `<body>` in the inspector, and throw the following error in the VSCode console:

`[Embedded Page] Uncaught (in promise) TypeError: Cannot read property 'focus' of null`

<img width="449" alt="Screen Shot 2020-01-16 at 11 55 03 AM" src="https://user-images.githubusercontent.com/2390265/72558614-e5fb8e00-3857-11ea-9ec1-6eded83f957d.png">

My launch.json looks like:

```
{
    "version": "0.2.0",
    "configurations": [
        {
            "type": "vscode-edge-devtools.debug",
            "request": "launch",
            "name": "Launch Edge against localhost",
            "url": "http://localhost:8000",
            "webRoot": "${workspaceFolder}/apps/xwars",
            "pathMapping": {
                "/": "${workspaceFolder}/apps/xwars"
            }
        }
    ]
}

**Comment by vidorteg** [comment]
Thanks everybody for the report:

It seems to me these can be treated as two separate issues:
- Link in stylesheets not opening the file (This issue)
- Path is not correct sometimes.(#118)

I have a fix for the link not opening in here: #119 , and will provide a patched version once it is checked in 👍

**Comment by vidorteg** [comment]
Sorry that It took some time, we just released the latest version (1.0.7) and this should be fixed.
